controllers/variables.go

```diff
@@ -0,0 +1,26 @@
+package controllers
+
+import (
+	"path"
+
+	"github.com/banzaicloud/k8s-objectmatcher/patch"
+
+	dcv1alpha1 "github.com/dominodatalab/distributed-compute-operator/api/v1alpha1"
+)
+
+var (
+	// DistributedComputeFinalizer is the custom identifier used to mark
+	// controller-managed resources that require pre-delete hook logic.
+	DistributedComputeFinalizer = path.Join(dcv1alpha1.GroupVersion.Group, "finalizer")
+
+	// PatchAnnotator applies state annotations to owned components.
+	PatchAnnotator = patch.NewAnnotator(path.Join(dcv1alpha1.GroupVersion.Group, "last-applied"))
```

Comment:
all shared by spark?

Reply:
Yeah, I looked at your work and figured that these are equivalent so I pulled this into its own file to indicate something that's available for the entire package